core/src/nl/tudelft/watchdog/core/logic/ui/listeners/staticanalysis/StaticAnalysisMessageClassifier.java

```diff
@@ -0,0 +1,31 @@
+package nl.tudelft.watchdog.core.logic.ui.listeners.staticanalysis;
+
+/**
+ * Holder for all {@link ClassificationBundle} that can contain messages.
+ * For each supported Plugin, 1 {@link ClassificationBundle} exists, with one bundle
+ * for the messages in the default IDE. These messages are obtained and processed
+ * in the respective subclass of {@link CoreMarkupModelListener}.
+ */
+public class StaticAnalysisMessageClassifier {
+
+    public static final String START_OF_CHECKSTYLE_MESSAGE = "Checkstyle: ";
```

Comment:
all public strings should be documented

Reply:
I feel like that would be mostly repeating the name of the variable. Added the documentation nonetheless